app/src/main/java/org/mozilla/fenix/ext/AppState.kt

```diff
@@ -17,6 +19,8 @@ import org.mozilla.fenix.home.recenttabs.RecentTab.SearchGroup
 
 @VisibleForTesting
 internal const val POCKET_STORIES_TO_SHOW_COUNT = 8
```

Comment:
Maybe a comment stating this count will include the sponsored stories would be useful. It is not obvious from the nearest code. (although it can be deduced from `combineRecommendedAndSponsoredStories` method)

Reply:
Thank you. Will address this in https://github.com/mozilla-mobile/fenix/pull/25037 which extracted this as a top level property.

Reply:
Seems to make more sense to add this comment here about the value including the count of sponsored stories, while previously just the naming could have been enough. Thank you!